sensor/test/sensor/sen0177/sensor-data-transformer.js

```diff
@@ -0,0 +1,43 @@
+let chai = require('chai');
+let expect = chai.expect;
+
+describe('SensorDataTransformer', () => {
+  const SensorDataTransformer = require('../../../services/sensor/sen0177/sensor-data-transformer');
+
+  it('Should properly convert integer values', (done) => {
```

Comment:
`done` is need only if U do async tests.
here it can be just
`it('Should properly convert integer values', () => {`
and you dont need to call `done()` on end